kolibri/core/tasks/storage.py

```diff
@@ -115,6 +116,17 @@ def __contains__(self, item):
                 is not None
             )
 
+    @staticmethod
+    def recreate_default_tables(engine):
+        """
+        Recreates the default tables for the job storage backend.
+        """
+        Base.metadata.drop_all(engine)
+        scheduledjobs_base = declarative_base()
+        scheduledjobs_table = Table("scheduledjobs", scheduledjobs_base.metadata)
+        scheduledjobs_table.drop(engine, checkfirst=True)
+        Base.metadata.create_all(engine)
```

Comment:
It feels like we should probably remove `self.Base.metadata.create_all(self.engine)` from the `__init__` function and defer to this. One possible issue with that is that we would have to add an invocation for this to all the tests, but if you're happy to do that, it seems worthwhile to move the logic wholly into this static method.

Reply:
Done, as you commented, several tests had to be changed. I'm afraid a session fixture in `conftest.py` didn't work due to the way `Storage` was initializated in the tests themselves i.e. The fixture worked but tests initalization redid the table, so this approach is more direct (but more verbose).

Reply:
I've reverted this change because it forced to do code modifications in a bunch of test files, creating a new pytest fixture and, even then, it produced problems when running it in postgresql. It seems it's not worth removing one line of logic if it makes us adding dozens of lines of code in tests.